tasks/cockpit-tasks-centosci.yaml

```diff
@@ -63,4 +63,4 @@ spec:
       - name: tmp
         emptyDir:
           medium: Memory
-          sizeLimit: 10G
+          sizeLimit: 14G
```

Comment:
Why do "nondestructive VMs pile up a lot of cruft in their overlays"? I guess there are stuff like journal and pcp that run the whole time, but I don't think that it should pile up a few GBs in the hour it runs.

Reply:
It's not in the file system. The overlays work on the  block layer, so if you create a 10 MB file, delete it, and do this a couple of times, file systems tend to allocate different blocks each. This reflects in the overlay, which over time will fill the entire virtual disk size. I think there's some techniques to compress them regularly, but I don't want to spend much time on that right now honestly.